torch_xla/distributed/cluster.py

```diff
@@ -382,6 +383,39 @@ def add_service_worker(tpu_name):
 
     return workers
 
+  def get_tpuvm_client_workers(self):
```

Comment:
this function is pretty similar to `get_client_workers`, can we maybe have `get_client_workers` only, and use `self._tpuvm_mode` as a switch to only differ when it's needed?

Reply:
hmm, this function is actually modify from the get_service_workers(given a tpu_name, return all of the gvm info). Let me see if there is a good way to integrate this with `get_client_workers`

Reply:
I don't see a very clean way of merging this to the `get_client_worker`, I could merge it with the `get_service_workers` but it will be weird. I think we can clean this up when we drop the support for the 2vm. At that time we only need to maintain one worker set (service-worker == client-worker).